app/types/commons/store.ts

```diff
@@ -11,4 +11,6 @@ export type YoutubeMeta = {
   title: string;
   thumbnail: string;
   alt: string;
+  artist: string;
+  tag: string;
```

Comment:
태그는 여러개가 붙을 수 있으니 tags가 좋아보입니다

```suggestion
  tags: string[];
```